lib/commands/config.ts

```diff
@@ -76,6 +76,14 @@ const command = {
 			return;
 		}
 		const config = ProjectConfig.getConfig(argv.global);
+
+		GoogleAnalytics.post({
+			cd: "Config",
```

Comment:
"Config" as screen view alone isn't too meaningful as you can't tell apart the get from the set.

Reply:
I guess the same applies to generate + template, though I'm ot sure what param can be used to specify sub command and args easily. Maybe further details can be attached via events as far as I can tell or just separate screen names if nothing else works.